RELEASE/scripts/sl_ascend/sl_test.ash

```diff
@@ -0,0 +1,60 @@
+script "sl_test.ash"
+
+import <sl_ascend.ash>
+
+boolean passed;
+
+void assertTrue(boolean val, string message)
+{
+	if (!val)
+	{
+		print("Assertion failed: " + message, "red");
+	}
+	passed &= val;
```

Comment:
I could easily be wrong because I've never used these operators in ASH, but isn't `&=` usually bitwise? I mean, it probably still works, just seems a little weird.

Reply:
```
> ash (true & true)

Returned: true

> ash (true & false)

Returned: false
```
It works, and alas, `&&=` isn't a thing.